core/src/main/resources/hudson/PluginManager/table.jelly

```diff
@@ -38,6 +38,7 @@ THE SOFTWARE.
   </st:documentation>
   <l:layout title="${%Updates} - ${%Plugin Manager}" permission="${app.SYSTEM_READ}">
     <st:include page="sidepanel.jelly"/>
+    <l:breadcrumb title="${%Updates}" />
```

Comment:
🐛 It's working for advanced, available, installed, but not for this one. 

⚠️ Due to other similar issues I found, I think the expected behavior is not working if the previous breadcrumb is also the current page, which leads to an inconsistent behavior for the end user.

<details><summary>Screenshot</summary>

### Before

![Screenshot_2022-11-29_192417_001](https://user-images.githubusercontent.com/2662497/204614547-e6ce1c11-c4db-484f-81d1-37f8dc836646.png)


### After

Not a link, but still a `li`

![Screenshot_2022-11-29_192456_001](https://user-images.githubusercontent.com/2662497/204614640-650ff449-dc90-4362-917a-499fb6832f9f.png)


</details>